tests/TestCase/Shell/CacheShellTest.php

```diff
@@ -34,7 +34,7 @@ public function setUp()
         parent::setUp();
         $this->io = $this->getMockBuilder('Cake\Console\ConsoleIo')->getMock();
         $this->shell = new CacheShell($this->io);
-        Cache::config('test', ['engine' => 'File', 'path' => TMP]);
+        Cache::config('test', ['engine' => 'File', 'path' => TMP . 'cache']);
```

Comment:
I think we should just be consistent everywhere, and the consensus was to always have trailing slashs for all paths (ROOT is the only exception right now in the whole system).